community/csv/src/main/java/org/neo4j/csv/reader/BufferedCharSeeker.java

```diff
@@ -250,6 +252,19 @@ private boolean eof( Mark mark )
         return false;
     }
 
+    private boolean getTrimStringIgnoreErrors( Configuration config, boolean defaultValue )
+    {
+        try
+        {
+            return config.trimStrings();
+        }
+        catch ( Throwable t )
+        {
+            System.err.println( "Error extracting 'trimStrings' setting from configuration: " + config );
```

Comment:
Very surrealistic, but I understand that we can't do much about it. Perhaps I'll just remove the System.err though. Cheers